src/main/java/domain/gamer/GamerName.java

```diff
@@ -0,0 +1,24 @@
+package domain.gamer;
+
+public class GamerName {
+
+    private static final int MIN_LENGTH = 1;
+    private static final int MAX_LENGTH = 20;
+
+    private final String value;
+
+    public GamerName(String value) {
+        validateLength(value);
+        this.value = value;
+    }
+
+    private void validateLength(String value) {
+        if (value.length() < MIN_LENGTH || value.length() > MAX_LENGTH) {
```

Comment:
미세 가독성 팁으로 꺽쇠 방향을 일치시키면 조금 더 잘 읽힙니다
```suggestion
        if (value.length() < MIN_LENGTH || MAX_LENGTH < value.length()) {
```